recipes/flammkuchen/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "flammkuchen" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e9aab9b229ace70d879b85618a9ce0e88dd6ce35d4dbcdfd60c6b61c33a1b4fb
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This recipe can be `noarch`, so please remove the `skip` line and add `noarch: python` instead, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.